README.md

```diff
@@ -168,15 +164,15 @@ how to programmatically update sets.
 
 ## Credits
 
-This project was originally created by Honko and is primarly maintained by Austin and pre.
+This project was created by Honko and is primarily maintained by Austin and pre.
```

Comment:
While this is being updated, can you please do me a favor and remove "and pre"? Thanks!